crates/sui-types/src/sui_sdk_types_conversions.rs

```diff
@@ -9,7 +9,25 @@
 //! directly to avoid going through the BCS machinery.
 
 use fastcrypto::traits::ToFromBytes;
-use sui_sdk_types::*;
+use sui_sdk_types::{
```

Comment:
Maybe undo this one?

Reply:
I intentionally did this because the module `bcs` in `sui_sdk_types` started to conflict with normal `bcs`